onepassword_rules/onepassword_sensitive_item_access.yml

```diff
@@ -0,0 +1,70 @@
+AnalysisType: rule
+Filename: onepassword_sensitive_item_access.py
+RuleID: OnePassword.Sensitive.Item
+DedupPeriodMinutes: 30
+DisplayName: Configuration Required - Sensitive 1Password Item Accessed
+Enabled: true
```

Comment:
Should this be disabled by default because it requires configuration?

Reply:
good catch, yes